src/redux/categories/categories.js

```diff
@@ -1,6 +1,6 @@
 // categories.js
```

Comment:
- [ ] In your [index.js](https://github.com/dqarias/bookstore/blob/redux-component/src/index.js), please make sure main React component is wrapped in Provider and assign store.

```
import store from './redux/configureStore';

    ReactDOM.render(
    <Provider store={store}>
        <App />
    </Provider>,
    document.getElementById('root')
    )
```

Reply:
I did it in App.js file